src/components/PhaseBanner.tsx

```diff
@@ -18,3 +18,5 @@ export const PhaseBanner: FunctionComponent<PhaseBannerProps> = ({
     </p>
   </div>
 );
+
+export default PhaseBanner;
```

Comment:
What are your thoughts when to `export const` inline and when to have `export default` at end of file?  There's a mix of both in the project at the moment (just curious, no need to change for this PR).